nixos/modules/misc/ids.nix

```diff
@@ -356,6 +356,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      synapse-diskspace-janitor = 400; # TODO: change to next sequential before merge!!!!!!!!!!!!!
```

Comment:
I don't think this is needed: it doesn't look like this service manages a lot of state files, so you should use systemd `DynamicUser` mechanism and let systemd deals with uid/gids.

Reply:
Yes, I tried that, but systemd gave us obscure errors for the `start` task:

```
Feb 25 03:37:48 bokkusu systemd[1]: matrix-synapse-diskspace-janitor.service: Failed to run 'start' task: Invalid argument
Feb 25 03:37:48 bokkusu systemd[1]: matrix-synapse-diskspace-janitor.service: Failed with result 'resources'.
```

Reply:
You can just create a user with users.users.